### DISCUSSION TITLE
Question: Is there any plan to support stable-diffusion-webui-forge?

### DISCUSSION THREAD

**Discussion by serick4126** [hako-mikan/sd-webui-lora-block-weight#148] [General]
I always use extensions, LBW and Negpip are a must in my workflow and I appreciate their development.
By the way, today a new WebUI fork called stable-diffusion-webui-forge was released by the author of controlnet.
https://github.com/lllyasviel/stable-diffusion-webui-forge

This fork uses less VRAM and faster inference speed compared to the A1111 WebUI, but currently LBW and Negpip do not work, probably due to the backend changes.
Is there any plan to support this fork in the future?

---
いつも拡張機能を使用しております、LBWおよびNegpipは私の環境において無くてはならない存在で、開発に感謝しています。
ところで、本日stable-diffusion-webui-forgeというWebUIフォークがcontrolnetの作者によりリリースされました。
https://github.com/lllyasviel/stable-diffusion-webui-forge

このフォークではA1111のWebUIと比較しVRAM使用量の低減や、推論速度の高速化が行われていますが、そのバックエンド変更のためか現在のところLBWならびにNegpipが動作しません。
今後こちらのフォークで動作するよう対応予定はありますでしょうか？

**Comment by hako-mikan** [comment]
どちらも対応できたとは思いますが、なにぶん検証が足りてないのでどこぞにバグがあるかもしれません。とりあえず使ってみて下さい。
